src/utils/exportSQL/mysql.js

```diff
@@ -3,13 +3,14 @@ import { parseDefault } from "./shared";
 import { dbToTypes } from "../../data/datatypes";
 
 export function toMySQL(diagram) {
+  console.log(diagram)
   return `${diagram.tables
     .map(
       (table) =>
         `CREATE TABLE \`${table.name}\` (\n${table.fields
           .map(
             (field) =>
-              `\t\`${field.name}\` ${field.type}${field.unsigned ? " UNSIGNED" : ""}${field.size !== undefined && field.size !== "" ? "(" + field.size + ")" : ""}${
+              `\t\`${field.name}\` ${field.type}${field.values ? "(" + field.values.map(value=>"'" + value + "'").join(", ")  + ")" : ""}${field.unsigned ? " UNSIGNED" : ""}${field.size !== undefined && field.size !== "" ? "(" + field.size + ")" : ""}${
```

Comment:
Also, please format. If you're on vscode you can use alt+shift+f

Reply:
ok, I'm using the default "typescript and javascript language features" formatter in vscode